bin/maf/gather_summaries

```diff
@@ -35,23 +35,41 @@ if __name__ == "__main__":
         help="Suffix for directories within which to find resultsDbs. "
         "Default is None, which searches for all potential MAF directories.",
     )
+    parser.add_argument(
+        "--to_pickle",
+        dest="to_pickle",
+        action="store_true",
+        help="Create a .pkl file, instead of the default CSV file.",
+    )
     args = parser.parse_args()
 
+    # Identify a subset of MAF directories if suffix is set
     if args.suffix is None:
         runDirs = None
     else:
         runDirs = glob.glob(f"*{args.suffix}")
 
+    # Create output file name if needed
+    if args.outfile is None:
+        if args.suffix is None:
+            outfile = "summary"
+        else:
+            outfile = f"{args.suffix.replace('_', '')}_summary"
+        if args.to_pickle:
+            outfile = outfile + ".pkl"
+        else:
+            outfile = outfile + ".csv"
+
     # Connect to resultsDbs and pull summary stats into a nice Dataframe
     rc = RunComparison(baseDir=args.baseDir, runDirs=runDirs)
     print(f"Found directories {rc.runDirs}")
     mdict = rc.buildMetricDict()
-    print("len mdict", len(mdict))
+    print(f"And found {len(mdict)} metrics.")
     rc.addSummaryStats(mdict)
 
     # Save summary statistics
-    if args.suffix is None:
-        outfile = "summary.csv"
+    if args.to_pickle:
+        rc.summaryStats.to_pickle(outfile)
```

Comment:
I think maybe we should stay away from using pickles. They aren't really portable, not even between different versions of python. One (but not the only) better alternative is HDF5, which is both more portable and also much faster to write and read (even from python).

Reply:
Let me check with @yoachim - I added the pickle option because that's what he was using somewhere else and I didn't want to remove it if it was needed.